memor/session.py

```diff
@@ -367,7 +367,7 @@ def _validate_extract_json(json_object: Union[str, Dict[str, Any]]) -> Dict[str,
                 loaded_obj = json.loads(json_object)
             else:
                 loaded_obj = json_object.copy()
-            result["title"] = loaded_obj["title"]
+            result["title"] = loaded_obj.get("title", None)
```

Comment:
`title` is a required parameter; if the config lacks it, we should raise an error, so revert this line to `loaded_obj["title"]`.